src/uu/cp/src/cp.rs

```diff
@@ -391,6 +389,7 @@ mod options {
 }
 
 #[cfg(unix)]
+#[allow(dead_code)]
 static PRESERVABLE_ATTRIBUTES: &[&str] = &[
```

Comment:
could you please add a comment?

Reply:
I am sorry that one is a mistake, thank you for pointing it out.